### ISSUE TITLE
Inside code, words that begins with a dollar sign are still processed by markdown.

### ISSUE THREAD

**Issue by fmaida** [HappenApps/Quiver#475]
I'm actually writing notes in markdown and I'm actually having problems when I write code that contains one or more word that begins with a dollar sign. Each word that begins with a dollar sign is written in italic and the dollar sign is removed. This is very annoying, especially when I have to write memos about linux commands.

For Example: 

```
> Be sure to add this to the .bash_profile file:
> 
> ~~~~sh
> PATH=$PATH:$HOME/gems/bin
> ~~~~
```

Is rendered this way:

PATH=_PATH_ :HOME/gems/bin

Have also a look at this image:

![schermata 2016-02-22 alle 18 17 52](https://cloud.githubusercontent.com/assets/605017/13226232/b153be36-d990-11e5-8bd9-3f1621365c4c.png)

**Comment by ylian** [comment]
My bad. I didn't realize that fenced code blocks can use `~~~` as well. I always used <code>```</code>.

As a workaround you have two options:
1. Use <code>```</code> for fenced code blocks
2. Turn off LaTeX support in Markdown cells (Preferences > Cells > Markdown Cell > Uncheck "Allow embedding LaTeX in Markdown). Make sure you're running 3.0.1.

I will fix this in 3.0.2.

**Comment by fmaida** [comment]
Thank you. I just disabled LaTeX support (I wan't going to use it anyway).
